File: internal/app/system/model/overview.go
package model

type Overview struct {
	LiveRoomCount      int     `json:"liveRoomCount"`      // 直播间总数
	RecordingRoomCount int     `json:"recordingRoomCount"` // 正在录制的直播间总数
	RecordTimeCount    float64 `json:"recordTimeCount"`    // 总录制时长（单位：时）
	UnreadMessageCount int     `json:"unreadMessageCount"` // 未读消息总数
	ParseMediaCount    int     `json:"parseMediaCount"`    // 已解析的媒体文件总数
}

File: web/src/types/overview.ts
export interface Overview {
    liveRoomCount: number;
    recordingRoomCount: number;
    recordTimeCount: number;
    unreadMessageCount: number;
    parseMediaCount: number;
}

export interface MonitorInfo {
    cpu: CpuInfo;
    mem: MemoryInfo;
    disk: DiskInfo;
}

export interface CpuInfo {
    cpu: number;
    cores: number;
    modelName: string;
    mhz: number;
    percent: number;
}

export interface MemoryInfo {
    total: number;
    used: number;
    available: number;
    usedPercent: number;
}

export interface DiskInfo {
    path: string;
    fstype: string;
    total: number;
    free: number;
    used: number;
    usedPercent: number;
}

export interface MetricsResult {
    data: Map<string, MetricsData>;
}

export interface MetricsData {
    platform: string;
    totalRequests: number;
    totalErrors: number;
    totalPercent: number;
    mainRequests: number;
    mainErrors: number;
    mainPercent: number;
}
